claim/gql_mutations.py

```diff
@@ -264,6 +264,11 @@ class ClaimInputType(OpenIMISMutation.Input):
     items = graphene.List(ClaimItemInputType, required=False)
     services = graphene.List(ClaimServiceInputType, required=False)
 
+    patient_outcome = graphene.String(max_length=250, required=False)
+    visit_reason = graphene.String(max_length=500, required=False)
+    inbound_referral_date = graphene.Date(required=False)
+    referral_id = graphene.Int(required=False)
```

Comment:
This is already done on dev branch, merge 2 weeks ago and done also on frontend : 

https://github.com/openimis/openimis-be-claim_py/pull/181

It's probably better to not use your version as we will have to maintain in your specific environment